src/sentry_scope.c

```diff
@@ -338,8 +338,10 @@ sentry__scope_apply_to_event(const sentry_scope_t *scope,
     sentry_value_decref(contexts);
 
     if (mode & SENTRY_SCOPE_BREADCRUMBS) {
-        PLACE_CLONED_VALUE("breadcrumbs",
-            sentry__value_ring_buffer_to_list(scope->breadcrumbs));
+        sentry_value_t l
+            = sentry__value_ring_buffer_to_list(scope->breadcrumbs);
+        PLACE_CLONED_VALUE("breadcrumbs", l);
+        sentry_value_decref(l);
```

Comment:
This is actually double cloning, right? You clone the contents of `scope->breadcrumbs` once via `sentry__value_ring_buffer_to_list` and then again in `PLACE_CLONED_VALUE`, the latter is no longer necessary, because `l` already contains the cloned values. You must set the `breadcrumbs` by key, but without a clone (i.e., `PLACE_VALUE`).